package-lock.json

```diff
@@ -31250,6 +31256,12 @@
           "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.15.tgz",
           "integrity": "sha512-8xOcRHvCjnocdS5cpwXQXVzmmh5e5+saE2QGoeQmbKmRS6J3VQppPOIt0MnmE+4xlZoumy0GPG0D0MVIQbNA1A==",
           "dev": true
+        },
+        "minimist": {
+          "version": "1.2.0",
```

Comment:
Same here, can try to run `npm dedupe`.

Reply:
Again here there are packages that require the exact version for example
`cypress@3.4.1 > minimist@1.2.0`